javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java

```diff
@@ -1703,15 +1703,19 @@ private static IType createTypeFromModelName(String modelName, JavaSettings sett
         return new ClassType.Builder().packageName(finalTypePackage).name(finalTypeName).build();
     }
 
-    private static MethodNamer resolveMethodNamer(ProxyMethod proxyMethod, ConvenienceApi convenienceApi, boolean isProtocolMethod) {
+    protected MethodNamer resolveMethodNamer(ProxyMethod proxyMethod, ConvenienceApi convenienceApi, boolean isProtocolMethod) {
         if (!isProtocolMethod && convenienceApi != null) {
             return new MethodNamer(SchemaUtil.getJavaName(convenienceApi));
         } else {
-            if (proxyMethod.isSync()) {
-                return new MethodNamer(proxyMethod.getBaseName());
-            }
-            return new MethodNamer(proxyMethod.getName());
+            return resolveMethodNamer(proxyMethod);
+        }
+    }
+
+    protected final MethodNamer resolveMethodNamer(ProxyMethod proxyMethod) {
+        if (proxyMethod.isSync()) {
+            return new MethodNamer(proxyMethod.getBaseName());
         }
+        return new MethodNamer(proxyMethod.getName());
```

Comment:
Fluent doesn't rely on convenienceApi.

Reply:
I remember rename on operation is done in `OperationNameNormalization`.

Do we rename it anywhere else?

Reply:
here: https://github.com/Azure/autorest.java/blob/8c72ed64834e22cfebcf49fab81b29d90bf9d5ee/javagen/src/main/java/com/azure/autorest/mapper/ClientMethodMapper.java#L754-L761

It happens after preprocessor(`OperationNameNormalization`).